### DISCUSSION TITLE
Continued issues with videos from YouTube

### DISCUSSION THREAD

**Discussion by Xdog42** [rocksdanister/lively#874] [Q&A]
Hi, I asked a question about this earlier, but I never got a clear or conscience answer. Whenever I link a video from youtube, it shows the banners and title and never loops properly. The last time I asked I got linked to another discussion but it never helped me figure out what's going on. Is this just an inescapable issue with videos that are linked from youtube or is there a way to get around this banner and loop issue? I've been trying to find a solution for a month and can't figure it out. Please help me out.
![lively](https://user-images.githubusercontent.com/93849895/140630325-3e9c6ae0-6efd-4e14-b0b0-756029d4d931.jpg)

**Comment by rocksdanister** [comment]
https://github.com/rocksdanister/lively/wiki/Streaming-Wallpaper#not-working
